src/openhome/client.js

```
@@ -0,0 +1,504 @@
+      case 'stop':
+        await callTransport('Stop');
+        break;
+      case 'next':
+        await callTransport('SkipNext');
+        break;
+      case 'previous':
+      case 'prev':
+        await callTransport('SkipPrevious');
+        break;
+      case 'vol_abs':
+        const volume = Math.max(0, Math.min(100, Number(value) || 0));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)